Author: Mark Stapp <mjs@cisco.com>
Description: tests: use little-endian order for libyang api
 Use the expected - little-endian - byte-order for a param
 to one of the libyang apis; tests fail on LE architectures
 otherwise.

diff -Naurp frr.orig/tests/lib/northbound/test_oper_data.c frr/tests/lib/northbound/test_oper_data.c
--- frr.orig/tests/lib/northbound/test_oper_data.c
+++ frr/tests/lib/northbound/test_oper_data.c
@@ -253,9 +253,10 @@ static enum nb_error frr_test_module_c2c
 							struct lyd_node *parent)
 {
 	const struct lysc_node *snode = nb_node->snode;
-	uint32_t value = 0xAB010203;
+	uint32_t value = htole32(0xAB010203);
 	LY_ERR err;
 
+	/* Note that this api expects 'value' to be in little-endian form */
 	err = lyd_new_term_bin(parent, snode->module, snode->name, &value, sizeof(value),
 			       LYD_NEW_PATH_UPDATE, NULL);
 	assert(err == LY_SUCCESS);
